src/README.md

```diff
@@ -0,0 +1,17 @@
+Collect data from StatsCan and other statistics agencies and convert to clean CSV files.
+
+
+### Data
```

Comment:
Once you've started documenting the datasets we are actually using more thoroughly, I think we maybe come back and update this section.

Reply:
Right. I have added a line for it.